lib/models/room.js

```diff
@@ -237,43 +316,226 @@ Room.prototype.getAvatarUrl = function(baseUrl, width, height, resizeMethod,
  };
 
 /**
- * Add some events to this room's timeline. Will fire "Room.timeline" for
- * each event added.
+ * Add a new timeline to this room
+ *
+ * @return {module:models/event-timeline~EventTimeline} newly-created timeline
+ */
+Room.prototype.addTimeline = function() {
```

Comment:
Feels weird that this doesn't accept a timeline, but I guess there is a good reason for it? When would the caller know enough to know that a new timeline is required yet not be able to make one themselves?